### DISCUSSION TITLE
Keycloak Theme

### DISCUSSION THREAD

**Discussion by tjcrone** [nebari-dev/nebari#1440] [Q&A] [answered]
Is there currently any way to customize or add themes to the Keycloak login page? What would be the right way to do this? Thanks!

**Comment by iameskild** [comment]
Hi @tjcrone, this seems possible but might not be entirely trivial. I have not attempted this and it will likely take some time to investigate. Let me bring it back to the team to see if they have any additional information :)

**Comment by trallard** [comment, marked as answer]
Hey, @tjcrone a quick search on Keycloak shows it is indeed possible to customise themes for Keycloak, see:

- https://www.keycloak.org/docs/latest/server_development/
- https://keycloak.discourse.group/t/keycloak-17-0-best-practices-for-theme-customization/15134

Is there anything, in particular, you are looking at customising?

**Reply by verhulstm** [reply]
I also want to be able to create a custom login page
